metric/unaggregated/types.go

```diff
@@ -119,14 +123,14 @@ func (m *MetricUnion) String() string {
 	}
 }
 
-// Reset resets the metric union
+// Reset resets the metric union.
 func (m *MetricUnion) Reset() { *m = emptyMetricUnion }
 
-// Counter returns the counter metric
+// Counter returns the counter metric.
 func (m *MetricUnion) Counter() Counter { return Counter{ID: m.ID, Value: m.CounterVal} }
 
-// BatchTimer returns the batch timer metric
+// BatchTimer returns the batch timer metric.
```

Comment:
Is it safe to return the slice of floats here even if it was allocated from a pool?